shared/rspec/support/sanitize.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
```

Comment:
the template structure was changed @junan :smile:

There is no `shared` folder anymore

![image](https://user-images.githubusercontent.com/11751745/90472582-058aca00-e14b-11ea-891a-fbef7dab76e8.png)